src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.RAND.cs

```diff
@@ -9,16 +9,15 @@ internal static partial class Interop
 {
     internal static partial class Crypto
     {
-        internal static unsafe bool GetRandomBytes(byte* pbBuffer, int count)
+        internal static bool GetRandomBytes(ref byte pbBuffer, int count)
         {
-            Debug.Assert(pbBuffer != null);
```

Comment:
As far as I know it's impossible to craft a `ref byte` in C# that is a null pointer (without resorting to unsafe code).